app/models/user.rb

```diff
@@ -43,13 +43,26 @@ def forget
   end
 
   def activate
-    update activated: true, activated_at: Time.zone.now
+    update_columns activated: true, activated_at: Time.zone.now
   end
 
   def send_activation_email
     UserMailer.account_activation(self).deliver_now
   end
 
+  def create_reset_digest
+    self.reset_token = User.new_token
+    update_columns reset_digest: User.digest(reset_token), reset_sent_at: Time.zone.now
```

Comment:
k dùng update_columns nhé, dùng update đi em

Reply:
Em dùng update nó toàn bị lỗi chị ạ @@

Reply:
![Screenshot from 2019-04-26 09-38-37](https://user-images.githubusercontent.com/45978973/56779865-25380980-6807-11e9-8b20-c888a5ae81b9.png)